jupyter_book/toc.py

```diff
@@ -201,7 +202,7 @@ def _content_path_to_yaml(path, root_path, split_char="_", add_titles=True):
         title = _filename_to_title(path.name, split_char=split_char)
 
     path_rel_root = path.relative_to(root_path)
```

Comment:
Maybe change this too, unless path is always in subfolder for `root_path`
```suggestion
    path_rel_root = Path(os.path.relpath(path, root_path))
```